src/js/components/Select/SelectContainer.js

```diff
@@ -397,7 +430,13 @@ const SelectContainer = forwardRef(
               name={name}
               onClear={onClear}
               onFocus={() => setActiveIndex(-1)}
-              onMouseOver={() => setActiveIndex(-1)}
+              onMouseOver={() => {
+                setActiveIndex(-1);
+                const clearButton = clearRef.current;
+                if (clearButton && clearButton.focus) {
+                  clearButton.focus();
```

Comment:
@jcfilben doing a programmatic focus here as a result of mousing over feels a little bit strange to me.

Reply:
But agree it is a little odd, maybe we can explore another way to prevent the above behavior